python/pyspark/sql/dataframe.py

```diff
@@ -1344,6 +1362,11 @@ def cov(self, col1, col2):
 
     cov.__doc__ = DataFrame.cov.__doc__
 
+    def freqItems(self, cols, support=1e-2):
```

Comment:
here -- if support=None by default, we only need to set the default value in one place.